apps/mobile/src/features/ImportPrivateKey/hooks/useImportPrivateKey.ts

```diff
@@ -29,7 +32,23 @@ export const useImportPrivateKey = () => {
     }
 
     try {
+      // Store the private key
       await storePrivateKey(wallet.address, privateKey)
+
+      // Create a delegate for this owner
+      try {
```

Comment:
We craete the delegate right after the user imoprts his private key. This way we don't have to bring up the biometrics prompt to access the private key.